### PR TITLE
Update Heimdall to queue scans for the default branch

### PR DESCRIPTION
**Pull request by Kamsiy**
## Description

This PR:
- Handle GraphQL errors in private GitHub organizations
- Delete the the default branch name when processing github repos
- Update Bitbucket query for bitbucket v2

## Motivation and Context

The branch field has to be `null` to trigger an Artemis scan on the default branch

## How Has This Been Tested?

tested locally and in a dev environment

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation change

## Checklist

- [x] My code follows conforms to the coding standards.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: orchestrator/lambdas/layers/heimdall_repos/heimdall_repos/github_utils.py [hunk 1]
```diff
@@ -244,55 +254,44 @@ def _is_repo_valid(self, repo: dict):
...
-            )
-
         branch_names, timestamps = self._get_branch_names(repo_name, branches)
         if self.scan_options.default_branch_only:
             branch_names = [default_branch]
             if default_branch not in timestamps:
                 timestamps[default_branch] = "1970-01-01T00:00:00Z"
 
         for branch in branch_names:
+            search_branch = branch
```
Comment (mdfleury-wbd) [inline orchestrator/lambdas/layers/heimdall_repos/heimdall_repos/github_utils.py:271]: I think a tertiary assignment would be easier to understand this

**Review by mdfleury-wbd — APPROVED** [review]
Nitpicking on some stuff

File: orchestrator/lambdas/layers/heimdall_orgs/heimdall_orgs/org_queue_private_github.py [hunk 2]
```diff
@@ -83,12 +83,14 @@ def _request_orgs(self, cursor=None) -> Union[dict, None]:
             log.info("Error retrieving orgs for %s", self.service)
             return None
 
+        response = response.json()
```
[the remark this answers is not in the document]
Comment (Kamsiy) [reply orchestrator/lambdas/layers/heimdall_orgs/heimdall_orgs/org_queue_private_github.py]: I agree, I've moved the response handling to the query_service

File: orchestrator/lambdas/layers/heimdall_repos/heimdall_repos/github_utils.py (hunk 1 above)
Comment (mdfleury-wbd) [reply orchestrator/lambdas/layers/heimdall_repos/heimdall_repos/github_utils.py:271]: Nevermind. Ternary is the correct word, and its not easier to understand.

**Review by mdfleury-wbd — APPROVED** [review]

**Pull request merged**